README-internal.md

```diff
@@ -20,6 +18,16 @@ There are some atypical development and release procedures in use with this repo
  2. Commit authors are rewritten to `Pantheon Automation <bot@getpantheon.com>` as a request from Product. The author
     names appear on the dashboard and this creates a more professional presentation.
 
+## Cutting a new release
+ 1. Update CHANGELOG.md. In the past, the copy has been created in consultation with Ari / product.
+ 1. Ensure the commit message for the last commit in this release says what we want to have appearing on the
+    dashboard as an available update. See [CORE-2258](https://getpantheon.atlassian.net/browse/CORE-2258) for
+    the inaugural example of such a commit message. All changes are committed to `pantheon-upstreams` as a single
+    commit, and the message that is used for it is the one from the last commit.
+    * Typically the CHANGELOG.md commit is the last one and so is the one whose commit message should be wordsmithed.
+ 1. Trigger the new release to `pantheon-upstreams` by `--ff-only`-merging `default` into `release` and pushing the 
```

Comment:
```suggestion
 3. Trigger the new release to `pantheon-upstreams` by `--ff-only`-merging `default` into `release` and pushing the 
```

Reply:
It would be cool if there were a make command or script to do this step. That would require filtering out said script during the release process. If you don't want to do that, it would be an improvement to type out the full git command instead of describing it. The character count it similar, so let's err on the side of specificity.